src/sass/auction.sass

```diff
@@ -73,3 +73,7 @@ $bid-price: #74BEE2
 
 .action-button
   min-width: 200px
+
+.is-button-styled
```

Comment:
is button styled does not really say what is behind it. So `is-button-bold` or so would have been better I think